flink-cdc-connect/flink-cdc-source-connectors/flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/TiKVRichParallelSourceFunction.java

```diff
@@ -101,12 +103,14 @@ public TiKVRichParallelSourceFunction(
             TiKVChangeEventDeserializationSchema<T> changeEventDeserializationSchema,
             TiConfiguration tiConf,
             StartupMode startupMode,
+            Long startupTimestamp,
             String database,
             String tableName) {
         this.snapshotEventDeserializationSchema = snapshotEventDeserializationSchema;
         this.changeEventDeserializationSchema = changeEventDeserializationSchema;
         this.tiConf = tiConf;
         this.startupMode = startupMode;
+        this.startupTimestamp = startupTimestamp;
```

Comment:
Thanks for your feature @cavalierfly , Could you add tests to cover your change ?